syncer/syncer.go

```diff
@@ -219,7 +177,7 @@ type Syncer struct {
 }
 
 // NewSyncer creates a new Syncer.
-func NewSyncer(cfg *config.SubTaskConfig) *Syncer {
+func NewSyncer(cfg *config.SubTaskConfig, enableRelay bool) *Syncer {
```

Comment:
How about adding `EnableRelay` index in `config.SubTaskConfig`? Then we needn't add enableRelay arg in function `NewSyncer`...

Reply:
refine in b8f0c9e